apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_state/states/SellerState3a.java

```diff
@@ -319,6 +337,13 @@ protected void onViewAttached() {
             qrCodeImageView.setImage(model.getSmallQrCodeImage());
 
             paymentProof.textProperty().bindBidirectional(model.getPaymentProof());
+            paymentProof.setValidators(new ValidatorBase() {
```

Comment:
Better to create a custom Validator class for reuse.

Reply:
I'll try to make one that encapsulates both usages then

Reply:
done, please have a look at the latest push

Reply:
> I'll try to make one that encapsulates both usages then

It is better to have one validator for one thing. UI controls can take multiple validators. BitcoinAddressValidator, BitcoinTransactionValidator,...